test/tests/LanguagePackTest.cpp

```diff
@@ -38,7 +34,7 @@ TEST_F(LanguagePackTest, create_mutable_id_1)
     ILanguagePack* lang = LanguagePackFactory::FromText(1, "STR_0000:\n");
     ASSERT_EQ(lang->GetId(), 1);
     ASSERT_EQ(lang->GetCount(), 1U);
-    ASSERT_STREQ(lang->GetString(0), nullptr);
+    ASSERT_STREQ(lang->GetString(0), "");
```

Comment:
For some reason this changed on Linux when I made this PR, but Windows version started the same. Needs some investigation.

Reply:
It will return nullptr because the string is empty and it seems this have been like this since 2017. Maybe there is something under ASSERT_STREQ that didn't work right.

Reply:
Hmm, good point. Perhaps its usage got changed

Reply:
```
error: Expected equality of these values:
  lang->GetString(0)
    Which is: NULL
  ""
[  FAILED  ] LanguagePackTest.create_mutable_id_1 (42 ms)
```